### ISSUE TITLE
MMN shows "test failed" while it is possible to connect to node

### ISSUE THREAD

**Issue by chompomonim** [mysteriumnetwork/node#2194]
I have been testing node `0x9e8b9fc27d4c23d65d33fc2684c8572c3c1d5cf1` during weekend. Both OpenVPN and WireGuard connections works. However, monitoring agent is not able to send any traffic and marks node as "failing".

We should recheck if there are no issues in monitoring agent logs. And if it is running the latest node version.

**Comment by soffokl** [comment]
OpenVPN checks looks OK from the monitoring agent. The problem is that MMN does not see the difference between OpenVPN and Wireguard. This should be resolved by https://github.com/mysteriumnetwork/my.mysterium.network/issues/10

Another problem is that wireguard proposal of this provider shows broken payment. Which is most likely because of service restarts from WebUI.

```
{
	"format": "service-proposal/v1",
	"id": 1,
	"payment_method": {
		"bytes": 0,
		"duration": 0,
		"price": {},
		"type": ""
	},
	"payment_method_type": "",
	"provider_contacts": [{
		"definition": {
			"broker_addresses": ["nats://testnet-broker.mysterium.network:4222"],
			"topic": "0x9e8b9fc27d4c23d65d33fc2684c8572c3c1d5cf1.wireguard"
		},
		"type": "nats/v1"
	}, {
		"definition": {
			"broker_addresses": ["nats://testnet-broker.mysterium.network:4222"]
		},
		"type": "nats/p2p/v1"
	}],
	"provider_id": "0x9e8b9fc27d4c23d65d33fc2684c8572c3c1d5cf1",
	"service_definition": {
		"location": {
			"asn": 2856,
			"city": "Scotland",
			"continent": "EU",
			"country": "GB",
			"isp": "BT",
			"node_type": "residential"
		},
		"location_originate": {
			"asn": 2856,
			"city": "Scotland",
			"continent": "EU",
			"country": "GB",
			"isp": "BT",
			"node_type": "residential"
		}
	}
```

**Comment by chompomonim** [comment]
Should be fixed with v0.32
